packages/components/src/chart/d3chart/chart.js

```diff
@@ -32,6 +32,8 @@ import { getColor } from './utils/color';
 import ChartTooltip from './utils/tooltip';
 import { selectionLimit } from '../constants';
 
+const isRTL = () => document.documentElement.dir === 'rtl';
```

Comment:
Should this go in `./utils` to keep things DRY?

Reply:
Thanks for taking a look @jeffstieler!

You mean moving that function to `d3chart/utils/index.js` or creating a utils file so it's usable in other components?

In Gutenberg, they seem to repeat that function in each component that uses it ([#](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/popover/utils.js#L7), [#](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/date-time/date.js#L16)).

Reply:
> You mean moving that function to `d3chart/utils/index.js` or creating a utils file so it's usable in other components?

I was thinking a new utils file.

> In Gutenberg, they seem to repeat that function in each component that uses it

Ah, interesting pattern to establish. Let's not consider this a blocker then.